packages/Discretization/test/PointCloudProblemGenerator/create_ascii_data_from_exodus.py

```diff
@@ -0,0 +1,90 @@
+############################################################################
+# Copyright (c) 2012-2017 by the DataTransferKit authors                   #
+# All rights reserved.                                                     #
+#                                                                          #
+# This file is part of the DataTransferKit library. DataTransferKit is     #
+# distributed under a BSD 3-clause license. For the licensing terms see    #
+# the LICENSE file in the top-level directory.                             #
+############################################################################
+
+# This script reads an exodus file and dumps its node coordinates and
+# connectivities of all elements in all element blocks into 2 seperate files
+# to be used by driver test code
+
+import netCDF4
+import numpy
+import sys
```

Comment:
We would need to discuss the decencies that are introduced here.  I just checked an our base image currently comes with a "plain" Python 2.7.12 and does **not** even have NumPy. As for netcdf4-python, we do have an install of the netCDF C library on the image, so we could in principle look into installing the Python interface but calling it directly from C++ is also an option to consider.

Reply:
(Python 3.5.2 is also there in our base image.)

Reply:
we might want to call netcdf directly - we wont have to update the image, it will reduce files, and let us work with exodus files directly